### PR TITLE
Wildlands flagpatches

### PR DESCRIPTION
**Pull request by 1Sparky1**
Adds flagpatches for each of the Wildlands Successor states to the game, with the FSF, Pact, SPG and SSMD ones being selectable from the PMCG loadout (For use by Wildlands Squadron members).

Sprites by Dessysalta, who also gets credit for the idea. I'm just a code monkey.

### CODE REVIEW

**Comment by 1Sparky1** [comment]
!review

**Comment by dessysalta** [comment]
spriter here, stay winning wildchads
image

**Review by DreamySkrell — APPROVED** [review]

**Review by Generalcamo — APPROVED** [review]

**Review by niennab — APPROVED** [review]

File: code/modules/clothing/suits/modular_armor.dm [hunk 1]
```diff
@@ -560,3 +560,115 @@
...
+	the aid of the Alliance. Now it has formed the Northern Solarian Reconstruction Mandate (NSRM), seeking to rebuild the war torn \
+	region."
+	icon_state = "flagpatch_pact"
+	item_state = "flagpatch_pact"
+
+/obj/item/clothing/accessory/flagpatch/sfa
+	name = "solarian fleet administration flagpatch"
+	desc = "A patch bearing the ensign of the Solarian Fleet Administration. It is a rare sight to see anyone \
+	loyal enough to the SFA to have one of these made."
+	desc = "Before the total collapse of Solarian authority in much of the Middle and Outer Rings, the Tenth Middle \
```
Comment (NonQueueingMatt) [inline code/modules/clothing/suits/modular_armor.dm:618]:
```suggestion
	desc_extended = "Before the total collapse of Solarian authority in much of the Middle and Outer Rings, the Tenth Middle \
```

**Review by NonQueueingMatt — DISMISSED** [review]

File: code/modules/clothing/suits/modular_armor.dm [hunk 2]
```diff
@@ -560,3 +560,115 @@
...
+	like many successor states, were formerly a Solarian fleet group, the Free Solarian Fleets did not rule over any \
+	planets or claim any greater goals beyond making money hand over fist as mercenaries in the Wildlands. They had \
+	an utter disinterest in conquering and holding territory of their own, instead opting to serve as a mercenary fleet \
+	for the highest bidder in the Wildlands. The group fought alongside the Middle Ring Shield Pact in the war in the \
+	Northern Wildlands."
+	icon_state = "flagpatch_fsf"
+	item_state = "flagpatch_fsf"
+
+/obj/item/clothing/accessory/flagpatch/league
+	name = "league of independent corporate-free systems flagpatch"
```
Comment (SleepyGemmy) [inline code/modules/clothing/suits/modular_armor.dm:581]:
```suggestion
	name = "\improper League of Independent Corporate-free Systems flagpatch"
```

File: code/modules/clothing/suits/modular_armor.dm [hunk 3]
```diff
@@ -560,3 +560,115 @@
...
+	desc = "A patch bearing the ensign of the League of Independent Corporate-Free Systems. Wearing this \
+	on a corporate vessel would probably be a bad idea."
+	desc_extended = "Many radical political changes followed the retreat of the Solarian Alliance from its \
+	Middle and Outer Rings. In most areas this political change was accompanied by a reckoning over what, \
+	exactly, had caused the once-great Solarian Alliance to slowly atrophy, decay, and then abandon the majority \
+	of its outlying territories. In the Human Wildlands close to the former Alliance Neutral Zone, a proposal \
+	regarding this emerged: that the Alliance had collapsed due to the meddling of corporations and if only the \
+	Alliance had stood against the forces of interstellar capitalism, it would have never collapsed. Thus from the \
+	ashes of the Alliance’s colonies emerged something truly unforeseen in the Orion Spur: a left-leaning, anti-corporate \
```
Comment (SleepyGemmy) [inline code/modules/clothing/suits/modular_armor.dm:590]:
```suggestion
	ashes of the Alliance's colonies emerged something truly unforeseen in the Orion Spur: a left-leaning, anti-corporate \
```

File: code/modules/clothing/suits/modular_armor.dm [hunk 4]
```diff
@@ -560,3 +560,115 @@
...
+	successor state to the Alliance in the region. After its defeat in the war in the Northern Wildlands at the hands of the \
+	Solarian Restoration Front, the former territories of the League were placed under the Northern Solarian Reconstruction \
+	Mandate (NSRM), seeking to rebuild the war torn region."
+	icon_state = "flagpatch_league"
+	item_state = "flagpatch_league"
+
+/obj/item/clothing/accessory/flagpatch/pact
+	name = "middle ring shield pact flagpatch"
```
Comment (SleepyGemmy) [inline code/modules/clothing/suits/modular_armor.dm:598]:
```suggestion
	name = "\improper Middle Ring Shield Pact flagpatch"
```

File: code/modules/clothing/suits/modular_armor.dm [hunk 5]
```diff
@@ -560,3 +560,115 @@
...
+	with a robust (if arguably megacorporate-owned) local bureaucracy, led to the Pact becoming an island of relative \
+	stability in the anarchy of the Northern Wildlands. The Pact was devastated by the war in the Northern Wildlands, being caught \
+	between both the Solarian Restoration Front and the League of Independent Corporate-Free Systems, but emerged victorious with \
+	the aid of the Alliance. Now it has formed the Northern Solarian Reconstruction Mandate (NSRM), seeking to rebuild the war torn \
+	region."
+	icon_state = "flagpatch_pact"
+	item_state = "flagpatch_pact"
+
+/obj/item/clothing/accessory/flagpatch/sfa
+	name = "solarian fleet administration flagpatch"
```
Comment (SleepyGemmy) [inline code/modules/clothing/suits/modular_armor.dm:615]:
```suggestion
	name = "\improper Solarian Fleet Administration flagpatch"
```

File: code/modules/clothing/suits/modular_armor.dm [hunk 6]
```diff
@@ -560,3 +560,115 @@
...
+	desc = "A patch bearing the ensign of the Solarian Fleet Administration. It is a rare sight to see anyone \
+	loyal enough to the SFA to have one of these made."
+	desc_extended = "Before the total collapse of Solarian authority in much of the Middle and Outer Rings, the Tenth Middle \
+	Ring Battlegroup under the command of Fleet Admiral F.R. Beauchamp was considered to be the absolute bottom of the \
+	Solarian Navy’s figurative barrel. With much of its enlisted being made up of prisoners conscripted or enlisted into \
```
Comment (SleepyGemmy) [inline code/modules/clothing/suits/modular_armor.dm:620]:
```suggestion
	Solarian Navy's figurative barrel. With much of its enlisted being made up of prisoners conscripted or enlisted into \
```

File: code/modules/clothing/suits/modular_armor.dm [hunk 7]
```diff
@@ -560,3 +560,115 @@
...
+	the Navy for a reduced prison sentence and the vast majority of its officers being washouts deliberately reassigned to the \
+	Tenth due to their failings, discipline was lax even before the phoron crisis. Despite doubts from many Solarian naval \
+	officers, the Solarian collapse was a time of many surprises, including the Tenth’s shocking rallying under Admiral \
```
Comment (SleepyGemmy) [inline code/modules/clothing/suits/modular_armor.dm:623]:
```suggestion
	officers, the Solarian collapse was a time of many surprises, including the Tenth's shocking rallying under Admiral \
```

File: code/modules/clothing/suits/modular_armor.dm [hunk 8]
```diff
@@ -560,3 +560,115 @@
...
+	Beauchamp to form the so-called Southern Fleet Administration shortly after the general Solarian withdraw from the region \
+	that would come to be known as the Southern Wildlands. The SFA were all but wiped out by the combined forces of the Southern \
+	Solarian Military District and the Solarian Provisional Government, and have since become a loose band of bandits and pirates \
+	operating in lawless regions of space throughout the Spur."
+	icon_state = "flagpatch_sfa"
+	item_state = "flagpatch_sfa"
+
+/obj/item/clothing/accessory/flagpatch/spg
+	name = "solarian provisional government flagpatch"
```
Comment (SleepyGemmy) [inline code/modules/clothing/suits/modular_armor.dm:632]:
```suggestion
	name = "\improper Solarian Provisional Government flagpatch"
```

File: code/modules/clothing/suits/modular_armor.dm [hunk 9]
```diff
@@ -560,3 +560,115 @@
...
+	desc = "A patch bearing the ensign of the Solarian Provisional Government, most commonly worn by the \
+	Provisional Government's law enforcement."
```
Comment (SleepyGemmy) [inline code/modules/clothing/suits/modular_armor.dm:634]:
```suggestion
	Provisional Government's law enforcement."
```

File: code/modules/clothing/suits/modular_armor.dm [hunk 10]
```diff
@@ -560,3 +560,115 @@
...
+	desc_extended = "In the Southern Human Wildlands lay the Solarian Provisional Government, the remnants of the Solarian \
+	Alliance’s massive bureaucratic apparatus that once lorded over the vast majority of the Middle Ring. While it was a mere rump \
```
Comment (SleepyGemmy) [inline code/modules/clothing/suits/modular_armor.dm:636]:
```suggestion
	Alliance's massive bureaucratic apparatus that once lorded over the vast majority of the Middle Ring. While it was a mere rump \
```

File: code/modules/clothing/suits/modular_armor.dm [hunk 11]
```diff
@@ -560,3 +560,115 @@
...
+	state compared to the titanic bureaucracy that once managed planets from Mictlan to New Hai Phong and most of its skilled bureaucrats \
+	fled to the what remained of the Alliance, the SPG managed to hold onto a small amount of ground near the Solarian border with the \
+	Wildlands thanks to the presence of the Sixth Middle Ring Battlegroup and unofficial support from the Alliance itself. The SPG aligned \
+	itself closely with the Alliance and the Alliance responded by offering “humanitarian” aid to the region under the SPG’s control. \
```
Comment (SleepyGemmy) [inline code/modules/clothing/suits/modular_armor.dm:640]:
```suggestion
	itself closely with the Alliance and the Alliance responded by offering “humanitarian” aid to the region under the SPG's control. \
```

File: code/modules/clothing/suits/modular_armor.dm [hunk 12]
```diff
@@ -560,3 +560,115 @@
...
+	In response the provisional government stated its clear intentions to rejoin the Alliance when the region around it stabilized. The \
+	Provisional Government has since merged with the Southern Solarian Military District to form the Southern Solarian Reconstruction \
+	Mandate (SSRM) shortly after the civil war in wake of the unrest and casualties that Szalai and MacPherson sought to undo."
+	icon_state = "flagpatch_spg"
+	item_state = "flagpatch_spg"
+
+/obj/item/clothing/accessory/flagpatch/srf
+	name = "solarian restoration front flagpatch"
```
Comment (SleepyGemmy) [inline code/modules/clothing/suits/modular_armor.dm:648]:
```suggestion
	name = "\improper Solarian Restoration Front flagpatch"
```

File: code/modules/clothing/suits/modular_armor.dm [hunk 13]
```diff
@@ -560,3 +560,115 @@
...
+	desc = "A patch bearing the ensign of the Solarian Restoration Front. Considering the near universal hatred \
+	of the Front across the Spur, wearing this would be a bold move."
+	desc_extended = "Extremely nationalistic even by the standards of the authoritarian Solarian Alliance, the Solarian \
+	Restoration Front is often rightfully viewed as the most militaristic of the Solarian successor states that jockied for control \
+	of the Wildlands that had made up the former Middle and Outer Rings of the Solarian Alliance. The Front believed that the Alliance’s \
```
Comment (SleepyGemmy) [inline code/modules/clothing/suits/modular_armor.dm:653]:
```suggestion
	of the Wildlands that had made up the former Middle and Outer Rings of the Solarian Alliance. The Front believed that the Alliance's \
```

File: code/modules/clothing/suits/modular_armor.dm [hunk 14]
```diff
@@ -560,3 +560,115 @@
...
+	third Tajara ban was both not enough and too soft on aliens in the Alliance, and they viewed this softness as the main cause for its collapse. \
+	Thus the Front strived to create a highly-militarized, extremely authoritarian, and human-only state in the Wildlands as a basis for purifying \
+	the Sol Alliance itself of degenerating alien influence, no matter the cost. Defeated by a surprise intervention in the Northern Wildlands by \
+	the Alliance, the former territories of the Front were placed under the Northern Solarian Reconstruction Mandate (NSRM), seeking to rebuild the war torn region."
+	icon_state = "flagpatch_srf"
+	item_state = "flagpatch_srf"
+
+/obj/item/clothing/accessory/flagpatch/ssmd
+	name = "southern solarian military district flagpatch"
```
Comment (SleepyGemmy) [inline code/modules/clothing/suits/modular_armor.dm:662]:
```suggestion
	name = "\improper Southern Solarian Military District flagpatch"
```

File: html/changelogs/wildlandsflagpatches.yml [hunk 15]
```diff
@@ -0,0 +1,7 @@
+author: Dessysalta (Sprites), Sparky_hotdog (Code)
```
Comment (SleepyGemmy) [inline html/changelogs/wildlandsflagpatches.yml:1]:
```suggestion
author: Dessysalta (sprites), Sparky_hotdog (code)
```

File: code/modules/client/preference_setup/loadout/items/factions.dm [hunk 16]
```diff
@@ -438,6 +438,23 @@
 	epmc_uniform_fr_med["Sekhmet Intergalactic first responder uniform"] = /obj/item/clothing/under/rank/medical/first_responder/pmc/sekh
 	gear_tweaks += new /datum/gear_tweak/path(epmc_uniform_fr_med)
 
+/datum/gear/faction/wildlands_flagpatches
+	display_name = "wildlands flagpatch selection"
```
[the remark this answers is not in the document]
Comment (1Sparky1) [reply code/modules/client/preference_setup/loadout/items/factions.dm:442]: I chose the singular flagpatch over the plural, in line with the regular flagpatch selection

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**